args4j/examples/SampleMain.java

```diff
@@ -65,7 +68,7 @@ public void doMain(String[] args) throws IOException {
             // after parsing arguments, you should check
             // if enough arguments are given.
             if( arguments.isEmpty() )
-                throw new CmdLineException(parser,"No argument is given");
+                throw new CmdLineException(parser, Messages.ILLEGAL_PATH, "No argument is given");
```

Comment:
Thanks for the info!

With a second look at the sample, I'd like to change my pull request to [this code](https://gist.github.com/KarlFish/7b380a3f6ec8a4d4efe0bb8f86e53c80). This new sample shows usage of the same method calls _(isEmpty, parseArgument, printExample)_, but doesn't throw a CmdLineException. It also fixes [issue 132](https://github.com/kohsuke/args4j/issues/132).

How can I help merge these changes? Should I close this request and open a new one?